### DISCUSSION TITLE
"Streams" indicator for a map difficulty that requires multitouch

### DISCUSSION THREAD

**Discussion by Hissymaster** [ppy/osu#27880] [Q&A]
This might be an indicator that appears in the top left corner.

**Comment by Walavouchey** [comment]
are you talking about 2B maps or stream maps? i have no clue

**Comment by Hissymaster** [comment]
I was sort of thinking it as a tag similar to the epilepsy warning... though as it could involve anything that requires touching more than one area at once, which is impossible on mouse or standard tablet... 
Okay... there would be a switch in the editor for "designed for tag team"... otherwise, any map that can't be beaten with a mouse or tablet alone. Will be counted as an "osu!stream" map.

**Comment by Hissymaster** [comment]
I'm not talking about 2B.
